apps/netbox/netbox/ptlsbox/netbox.yaml

```diff
@@ -14,4 +14,9 @@ spec:
         name: hmctspublic
         namespace: flux-system
   releaseName: netbox
-# using default values from chart
\ No newline at end of file
+  values:
+    ingress:
+      hosts:
+        - host: netbox-sandbox.hmcts.net
```

Comment:
You'll need to add this private DNS record to point to ss ptslbox traefik

Reply:
https://github.com/hmcts/azure-private-dns/pull/685